### DISCUSSION TITLE
Is it possible to personalize the actions for different notifications?

### DISCUSSION THREAD

**Discussion by ComptonAlvaro** [thudugala/Plugin.LocalNotification#418] [Q&A]
I would like to set some actions in the notification in a MAUI application. This is the documentation: [Actions documentation](https://github.com/thudugala/Plugin.LocalNotification/wiki/Notification-with-Action)

I see that for a MAUI application, the acciones are set in the MAUIProgram.cs file. Does it means that for all the notifications I will have the same actions?

I would like to publish notificacions with different actions, dependen on the situation.

Thanks.

**Comment by thudugala** [comment]
@ComptonAlvaro, you can change NotificationCategoryType and set up different combinations of actions buttons
